### ISSUE TITLE
Can't override on the fly

### ISSUE THREAD

**Issue by garretyoder** [negusoft/GreenMatter#10]
Beautiful library, overriding theme colors works fine from the overridePalette function, but you can't override on the fly, eg loading theme colors from SharedPreferences as the Activity starts up.

**Comment by blurkidi** [comment]
No, you need to reload the activity for the change to take effect. If you have a look at the sample app, it calls recreate() when the overriding color changes:

 @Override
    protected void onActivityResult(int requestCode, int resultCode, Intent data) {
        if (resultCode == Activity.RESULT_OK) {
            recreate();
        }
        super.onActivityResult(requestCode, resultCode, data);
    }

I hope that this solution works for your.
